glue/plugins/scatter_group/__init__.py

```diff
@@ -0,0 +1,10 @@
+def setup():
+    from ...logger import logger
```

Comment:
Yes, I changed it when setting up the lazy loading of modules - the issue I ran into is that I think we really only should have one logger for the whole application otherwise I think it's hard to set the logging level globally - or am I wrong?

In other packages, such as Astropy, we have a single logger but it's still able to display in what sub-pacakge the message occurred. Are there other reasons we might want individual loggers?